telescope/query_test.py

```diff
@@ -148,15 +154,58 @@ def test_ndt_download_throughput_query_full_month(self):
        PARSE_IP(web100_log_entry.connection_spec.remote_ip) BETWEEN 35 AND 80)"""
     self.assertQueriesEqual(query_expected, query_actual)
 
+
+  def test_ndt_download_throughput_query_country_property(self):
+    start_time = datetime.datetime(2014, 1, 1)
+    end_time = datetime.datetime(2014, 2, 1)
+    server_ips = ['1.1.1.1', '2.2.2.2']
+    client_ips = None
+    client_country = "us"
+    query_actual = self.generate_download_throughput_query(start_time,
+                                                           end_time,
+                                                           server_ips,
+                                                           client_ips,
+                                                           client_country)
+    query_expected = """
+SELECT
+  connection_spec.data_direction,
+  web100_log_entry.log_time,
+  web100_log_entry.snap.CongSignals,
+  web100_log_entry.snap.HCThruOctetsAcked,
+  web100_log_entry.snap.SndLimTimeCwnd,
+  web100_log_entry.snap.SndLimTimeRwin,
+  web100_log_entry.snap.SndLimTimeSnd,
+  web100_log_entry.snap.State
+FROM
+  [plx.google:m_lab.2014_01.all]
+WHERE
+  connection_spec.data_direction IS NOT NULL
+  AND web100_log_entry.is_last_entry IS NOT NULL
+  AND web100_log_entry.snap.HCThruOctetsAcked IS NOT NULL
+  AND web100_log_entry.snap.CongSignals IS NOT NULL
+  AND web100_log_entry.connection_spec.remote_ip IS NOT NULL
+  AND web100_log_entry.connection_spec.local_ip IS NOT NULL
+  AND project = 0
+  AND web100_log_entry.is_last_entry = True
+  AND connection_spec.data_direction == 1
+  AND ((web100_log_entry.log_time >= 1388534400) AND (web100_log_entry.log_time < 1391212800))
+  AND (web100_log_entry.connection_spec.local_ip = '1.1.1.1' OR
+       web100_log_entry.connection_spec.local_ip = '2.2.2.2')
+  AND (connection_spec.client_geolocation.country_code = UPPER('us'))
```

Comment:
This test is failing now right? In case you don't already have it set up, I highly recommend setting up the pre-commit hook so that you don't have to remember to run unit tests manually before committing:
https://github.com/m-lab-tools/telescope/wiki/Running-unit-tests